src/Adapter/Security/Admin.php

```diff
@@ -59,7 +59,7 @@ class Admin
      */
     private $userProvider;
 
-    public function __construct(LegacyContext $context, TokenStorage $securityTokenStorage, UserProviderInterface $userProvider)
+    public function __construct(LegacyContext $context, TokenStorageInterface $securityTokenStorage, UserProviderInterface $userProvider)
```

Comment:
BC Break

Reply:
Are you sure? As `TokenStorage` is an implementation of `TokenStorageInterface`, the constructor can accept a `TokenStorage` 🤔

Reply:
But the change type of the constructor has impact on users of this class. So we need to document the BC Break.

Reply:
Just discussed with @PierreRambaud, and you're right. Sorry.